contents.json

```diff
@@ -4094,6 +4094,12 @@
       "category": "validation",
       "description": "Drop in user input validation for your iOS apps.",
       "homepage": "https://github.com/adamwaite/Validator"
+    }, {
+      "title": "ValidatedPropertyKit",
+      "category": "validation",
+      "description": "Easily validate your Properties with Property Wrappers 👮",
```

Comment:
@SvenTiigi could you add a full stop/period for description?

Reply:
Fixed with https://github.com/matteocrippa/awesome-swift/pull/1487/commits/7039b51a5a8796cdf6aa1512512231d18934c357 👌